doc/binder/environment.yml

```diff
@@ -0,0 +1,17 @@
+name: mne_bids
+channels:
+- defaults
+dependencies:
+- python<3.7
+- pip=19
+- mkl
+- numpy>=1.14
+- scipy>=0.18.1
+- scikit-learn
+- matplotlib
+- pip:
+  - mne>=0.19.1
+  - mne-bids
+  - nibabel
+  - pybv
+  - nilearn
```

Comment:
instead of providing a new file, could we just have a file that maintains the "diff"? The sphinx gallery config can take a list of dependency files as far as I know.

Reply:
see solution in https://github.com/mne-tools/mne-bids/pull/300#issuecomment-558547866